DataFormats/TrackTfGraph/plugins/BuildFile.xml

```diff
@@ -0,0 +1,11 @@
+<use name="FWCore/Framework"/>
+<use name="FWCore/ParameterSet"/>
+<use name="FWCore/PluginManager"/>
+<use name="PhysicsTools/TensorFlow"/>
+<use name="FWCore/Utilities"/>
+<use name="RecoTracker/Record"/>
```

Comment:
hi @hajohajo  - these dependencies on RecoTracker and PhysicsTools are not allowed in DataFormats.

Reply:
I think you should move TrackTfGraphDefProducer, TfGraphDefWrapper to RecoTracker/FinalTrackSelectors.